Source/Core/Core/IOS/Network/Socket.cpp

```diff
@@ -50,7 +50,7 @@ char* WiiSockMan::DecodeError(s32 ErrorCode)
 #endif
 }
 
-static s32 TranslateErrorCode(s32 native_error, bool isRW)
+static s32 TranslateErrorCode(s32 native_error, bool is_rw)
```

Comment:
I know nothing about this code.  Is `is_rw` mean "read"/"write"?

Reply:
It refers to read/write or rather `send(to)` and `recv(from)`.

The following functions can return
 - **EAGAIN**/**EWOULDBLOCK**:  `send(to)`, `recv(from)`, `accept`
 - **EINPROGRESS**: `connect`, `bind`
 - **WSAEWOULDBLOCK**: `send(to)`, `recv(from)`, `accept`, `connect`

It is used below to correct windows error codes **WSAEWOULDBLOCK** as it can be used in functions which don't return **EAGAIN**/**EWOULDBLOCK** but **EINPROGRESS** on Linux. Basically, the Linux code path will always return `-SO_EAGAIN`


I can add a comment about it if you want.

Reply:
I think it might be more the naming than the usage. Not too sure if spelling out the words would help tho...